test/libraries/TickBitmap.t.sol

```diff
@@ -288,9 +301,17 @@ contract TickBitmapTest is Test, GasSnapshot {
         }
     }
 
+    function isInitialized(int24 tick, int24 tickSpacing) internal view returns (bool) {
+        unchecked {
+            int24 compressed = tick / tickSpacing;
+            if (tick < 0 && tick % tickSpacing != 0) compressed--; // round towards negative infinity
```

Comment:
why do you have to subtract 1? why cant it just be the solidity that was previously in `TickBitmap`?
```
(int16 wordPos, uint8 bitPos) = position(tick / tickSpacing);
```

Reply:
That's the first two lines of `nextInitializedTickWithinOneWord`, which I factored out to a `compress` function in #654.

Reply:
but for `flipTick` theres a requirement that (tick % tickSpacing != 0) otherwise it reverts, so this line feels out of place here? Not sure what I'm missing

Reply:
Do you mean `isInitialized` should return `false` instead of rounding if `tick % tickSpacing != 0`?